src/clj/game/cards/programs.clj

```diff
@@ -144,6 +144,26 @@
                    :msg (msg "install " (:title target))
                    :effect (req (when (can-pay? state side nil :credit (:cost target))
                                   (runner-install state side target)))}]})
+   "Consume"
+   {:events {:runner-trash {:req (req (= (:side target) "Corp"))
+                              :optional
+                              {:prompt "Place a virus counter on Consume?"
+                               :yes-ability
+                               {:effect (effect (add-counter :runner card :virus 1)
```

Comment:
I think this will always have `side` as `:runner` but I could be wrong. Specifying the side is always safer.

Reply:
Are you saying it might not be necessary, but it's fine as-is?